src/api/dataSources.js

```diff
@@ -14,7 +12,7 @@ export async function stockxLowestPrice (item, filter) {
       ship_to: filter.country
     })
     const response = await fetch(request.url, request.headers)
-    if (!response.ok) return {}
+    if (!response.ok) return []
```

Comment:
Are we sure we want an empty list and not an empty object here?

Reply:
fixed